README.md

```diff
@@ -1,2 +1,24 @@
-# ms-office-macro-nuke
-Tool to Purge and Remove all Macro And Dynamic Content from MS Office File
+![MS Office Macro Bleach](docs/images/bleach.jpg)
+
+
+# MS Office Macro Bleach
+
+A command-line tool designed to detect and purge any and all macros and dynamic content from MS Office files.
+
+It currently supports '.docm' and'.docx' files and will continue to expand its file support with every released version. The next file types to be supported will be '.pdf'' and '.doc'.
+
+
+## Problem
+
+VBA and OLE content in MS Office files can, and have sometimes been made to, act as vehicles for malware delivery.
+
+Microsoft has previously attempted to protect users from macros by disabling them by default.  However, anybody is able to enable macros in an MS Office file before sending them on to a potential victim.
+
```

Comment:
I would remove this line and change it for something like. However, users can reenable it, or an Adversary with an initial foothold would be able to enable it again. For a large amount of environments (Schools, Controlled Environments, Corporations, Manufacturing) the Administrator might decide to remove all Dynamic Content of files going through Emails Pipelines, or in Conjunction with Web Gateways. Another example use case could be to Bleach Files submitted to HR by applicants through an HR portal.

Reply:
Should I just include 'However, users can reenable it, or an Adversary with an initial foothold would be able to enable it again' or the whole paragraph?

Reply:
Yeah I would remobe that line